code/xcvm/cosmwasm/contracts/escrow/Cargo.toml

```diff
@@ -18,8 +18,10 @@ cosmwasm-std = { workspace = true, features = ["ibc3"] }
 cw-storage-plus.workspace = true
 cw-utils = { workspace = true, features = ["std"] }
 cw2.workspace = true
+cw20.workspace = true
 parity-scale-codec.workspace = true
+serde-json-wasm.workspace = true
 strum.workspace = true
 thiserror.workspace = true
 
-xc-core = { path = "../../../lib/core", features = ["std"] }
+xc-core = { path = "../../../lib/core", features = ["std", "cw20"] }
```

Comment:
i see that contacts outside do not make it feature, it just cw20 and native all the time.

Reply:
The reason for separate function is that to have cw20 definitions in xc-core it needs to depend on cw20 crate which is not no_std.  I suppose I could just enable cw20 when std is selected.  Having cw20 as separate feature just thought cleaner I guess?  Thoughts?

Reply:
https://github.com/CosmWasm/cw-plus/pull/874

Reply:
Right, but that’s cw-plus. cw20 still pulls std in. We really probably should have a `cosmwasm` feature on `xc-core` crate since it should work on any chain. Let’s leave `cw20` for now and see how it goes.

Reply:
ah, yes. features are good, right.